MPI init is required before the help menu is allowed to find the process rank.
Ref. https://github.com/VI4IO/pfind/issues/17
Author: micketeer@gmail.com
--- src/pfind-main.c.orig	2022-06-13 22:07:24.979936591 +0200
+++ src/pfind-main.c	2022-06-13 22:06:50.856611055 +0200
@@ -31,6 +31,7 @@
 }
 
 int main(int argc, char ** argv){
+  MPI_Init(&argc, &argv);
   // output help with --help to enable running without mpiexec
   for(int i=0; i < argc; i++){
     if (strcmp(argv[i], "--help") == 0){
@@ -41,7 +42,6 @@
     }
   }
 
-  MPI_Init(&argc, &argv);
 
   pfind_options_t * options = pfind_parse_args(argc, argv, 0, MPI_COMM_WORLD);
 
